docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,284 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+NGINX Ingress Controller can now be used as the ingress controller for applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX IC provides on Istio-based environments without resorting to any workarounds. 
+
+
+Prior to 1.11 release, a configuration below would send two host headers to the backend; 

```

Comment:
I don't understand that this section has to do with NIC + Istio.  It seems to describe a deficiency but does not really have any context.
What I would expect is some type of introduction paragraph that describes the scenario - like begins below.

Reply:
I will reword it a bit. Something like:
`This walkthrough, will show you how you can use NGINX Ingress Controller with Istio service mesh....`